tests/agent/test_prompt.py

```
@@ -0,0 +1,250 @@
+        response = client.chat.completions.create(
+            model=model_name,
+            messages=[
+                {"role": "system", "content": ELEMENTWISE_ADD_PROMPT},
+                {"role": "user", "content": FFI_PROMPT_SIMPLE},
+            ],
+        )
+
+    return response.choices[0].message.content
+
+
+def call_anthropic_model(model_name: str, api_key: str, prompt: str) -> str:
```

Ruff (0.14.10)

Unused function argument: `prompt`

(ARG001)